src/client.rs

```diff
@@ -1,7 +1,8 @@
 use crate::environment::ApiEnvironment;
 use crate::services::{
     AccountBalanceBuilder, B2bBuilder, B2cBuilder, C2bRegisterBuilder, C2bSimulateBuilder,
-    MpesaExpressRequestBuilder, TransactionReversalBuilder, TransactionStatusBuilder,
+    DynamicQRBuilder, MpesaExpressRequestBuilder, TransactionReversalBuilder,
```

Comment:
The reason for the clippy error below is because the `dynamic_qr` cargo feature is not yet added to the default features